packages/cli/src/platform/PlatformCommand.ts

```diff
@@ -0,0 +1,22 @@
+// import { Command as MoltCommand } from '@molt/command'
+import { Command, unknownCommand } from '@prisma/internals'
+
+export class PlatformCommand implements Command {
+  public static new(): PlatformCommand {
+    return new PlatformCommand()
+  }
+
+  public async parse(argv: string[]) {
+    const isHasEarlyAccessFeatureFlag = Boolean(argv.find((_) => _.match(/early-access-feature/)))
+    if (!isHasEarlyAccessFeatureFlag) return unknownCommand('You must pass --early-access-feature to use platform commands.', 'platform') // prettier-ignore
```

Comment:
This is not the intended usage for this, we have an error for this case already here "EarlyAccessFlagError", it can be modified if needed.

Reply:
Thanks @Jolg42 found thanks to auto import!